### PR TITLE
Add support for Flood for Transmission

### PR DESCRIPTION
**Pull request by johman10**
**Description of the change**

These changes allow to run [Flood for Transmission ](https://github.com/johman10/flood-for-transmission) within the docker container. This can be done by using the following command:
`docker run -e "TRANSMISSION_WEB_HOME=/flood-for-transmission" -p 9091:9091 transmission`

The default behaviour hasn't changed in any way.

**Benefits**

More beautiful UI for everyone

**Possible drawbacks**

Bigger container size (though little)
Complexity (though little)

**Applicable issues**

- fixes https://github.com/johman10/flood-for-transmission/issues/383

**Additional information**

I'm the creator of Flood for Transmission. If you have any questions, let me know.

### CODE REVIEW

File: apps/transmission/Dockerfile [hunk 1]
```diff
@@ -47,9 +47,6 @@ RUN \
 # hadolint ignore=DL3007
 FROM ghcr.io/k8s-at-home/ubuntu:latest
 
-ARG TARGETPLATFORM
-ARG VERSION
```
Comment (johman10) [inline apps/transmission/Dockerfile:51]: These ARGS are already defined on the first lines.

Also note: TARGETPLATFORM and TARGETVERSION are never really used within the Dockerfile.

**Comment by bjw-s** [comment]
Thanks for creating this PR!
I can't promise we'll merge this since we also have a Flood Helm chart (https://github.com/k8s-at-home/charts/tree/master/charts/stable/flood) which could be used instead, even though would be a more lightweight to add it.

I'll have a chat about this with the other maintainers, and we will get back to you on it :)

**Comment by onedr0p** [comment]
Our images are really catered to Kubernetes and the Helm chart already mentions how to add a custom web-ui to Transmission.

https://github.com/k8s-at-home/charts/blob/master/charts/stable/transmission/values.yaml#L11-L24

**Comment by johman10** [comment]
I see! I'm sorry, I'm not all that familiar with your project, just contributing since I received an issue on my repo and figured why not give it a shot. :)

Feel free to close this if you feel this is out of place, no hard feelings! 👍

**Comment by onedr0p** [comment]
No problems, Kubernetes has a lot of advanced methods of dealing with things like this compared to other container orchestration platforms.

**Pull request closed without merging**